### ISSUE TITLE
Publishing front-end JavaScript collateral

### ISSUE THREAD

**Issue by jagoda** [bandwidthcom/bandwidth-webrtc#69]
@fuchsnj has indicated that currently this project is published to a CDN via S3. While this is all well and good, I propose that all front-end JavaScript projects should also be published to NPM and Bower. All three locations should be published to automatically and simultaneously based on tagged commits via Travis.

`webrtc-client` is already taken in NPM. What do we think of changing the name of this project to `bandwidth-webrtc` as a part of this work?

**Comment by jagoda** [comment]
@bandwidthcom/javascript

**Comment by fuchsnj** [comment]
I'm going to rename to bandwidth-webrtc in a couple hours unless someone says something.
@wtcross @jagoda @jasommerset

**Comment by wtcross** [comment]
I don't think `webrtc` should be in the name.

**Comment by wtcross** [comment]
That said, I can't think of an alternative that isn't lame sounding. :)

**Comment by wtcross** [comment]
How about `bandwidth-phone`?

**Comment by fuchsnj** [comment]
what about just `bandwidth-client`?

**Comment by nguyer** [comment]
I think `bandwidth-client` is far too generic. We have lots of clients. We have Android apps that are API clients. We have Node apps that are clients, etc, etc.

**Comment by jagoda** [comment]
I worry that `bandwidth-client` is ambiguous since we also have `node-bandwidth` and `bandwidth-iris` which are both "clients" for different systems.

**Comment by fuchsnj** [comment]
so is everyone ok with `bandwidth-phone` then?

**Comment by jagoda** [comment]
Fine with me.

**Comment by EdHintz** [comment]
I am not ok with bandwidth-phone because webrtc is more than phone calls.

**Comment by wtcross** [comment]
What if this grows beyond webrtc?

**Comment by wtcross** [comment]
Honestly this library has no business growing beyond a phone, in my opinion.

**Comment by wtcross** [comment]
I say that since it is built on SIP.js

**Comment by joeldodson** [comment]
+1 to Ed's point.  WebRTC (as I'm sure everyone here knows) isn't just about phone calls.  And the usage of SIP in WebRTC should be thought of only as a means for exchanging SDPs and managing sessions established between endpoints (going back to the roots of SIP).  
That said, I don't have a good name alternative.  bandwidth-webrtc sounds fine to me.

**Comment by wtcross** [comment]
@joeldodson `bandwidth-webrtc` has been chosen as the name.

I was pushing for another name because I don't think this library is positioned to be anything more than a phone. I'm not sure why it would want to be. Video maybe? I just don't like naming it after underlying tech. I wouldn't want to call our android client `bandwidth-pjsip`

**Comment by wtcross** [comment]
`bandwidth-webrtc` was decided as the name. Closing.

**Comment by wtcross** [comment]
@joeldodson I don't think this library is about webrtc, though. That was the basis of my suggestion. In my opinion this code base should not become anything more than a well tested abstraction and configuration layer on top of SIP.js

**Comment by wtcross** [comment]
That said, moving on :)

**Comment by wtcross** [comment]
I should not have closed this.

**Comment by joeldodson** [comment]
Thanks @wtcross, makes sense.  And I agree naming it based on webrtc is limiting.  And I also agree bandwidth-client is too general.  Seems like we're faced with the November conundrum, pick the best of the somewhat undesirable options.  

And sorry to be late to the party on this, I really just wanted to +1 @EdHintz that we should always be thinking of webrtc as more than just a phone, which I'm sure everyone here understands.

**Comment by wtcross** [comment]
Yes, I agree on how to think about webrtc. I try to discourage using webrtc to describe things, since it's an implementation detail.
